### DISCUSSION TITLE
Invitation to Participate in the 2024 MDAnalysis Community Survey

### DISCUSSION THREAD

**Discussion by jennaswa** [MDAnalysis/mdanalysis#4646] [Announcements]
We're launching a survey to gather your valuable feedback on MDAnalysis communication channels and resources as a part of an [Outreachy project](https://www.mdanalysis.org/2024/05/27/gsoc-outreachy-students/#adetutu-oluwasanmi-develop-a-communications-strategy-for-a-growing-mdanalysis-user-and-contributor-base).

The anonymous survey takes about 10 minutes, and your responses will help us foster an inclusive and supportive community.

Kindly start the survey using this link: [MDAnalysis Community Survey 2024](https://docs.google.com/forms/d/e/1FAIpQLSeTn6zIm-LIry7aRJ0IF_r_B4m4E0ddUEVIHxXGD-H3n-nOiQ/viewform?usp=sf_link). The deadline to complete the survey is **August 31st, 2024**.

You can learn more about the survey in our [latest blog post](https://www.mdanalysis.org/2024/07/31/survey-announcement/).

Thank you for your participation!

**Comment by adetutudeborah** [comment]
Hello Everyone,

Just a quick reminder about the [MDAnalysis Community Survey 2024](<https://docs.google.com/forms/d/e/1FAIpQLSeTn6zIm-LIry7aRJ0IF_r_B4m4E0ddUEVIHxXGD-H3n-nOiQ/viewform?usp=sf_link>). If you haven't already, please take a few minutes to complete the survey: [Survey Link](<https://docs.google.com/forms/d/e/1FAIpQLSeTn6zIm-LIry7aRJ0IF_r_B4m4E0ddUEVIHxXGD-H3n-nOiQ/viewform?usp=sf_link>).

Thank you for your time and contributions!
